packages/mint-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/mint-components",
-  "version": "1.4.0",
```

Comment:
Flagging that 1.4.1 is also being used by https://github.com/saasquatch/program-tools/pull/246

Reply:
ah was wondering what was deployed to that version, thank you!